packages/terra-worklist-data-grid/tests/wdio/worklist-data-grid-spec.js

```diff
@@ -28,98 +28,95 @@ Terra.describeViewports('WorklistDataGrid', ['medium', 'large'], () => {
       browser.keys(['Tab']
-        .concat(new Array(1).fill('ArrowDown'))
-        .concat(new Array(3).fill('ArrowUp'))
-        .concat(new Array(3).fill('ArrowLeft')));
+        .concat(new Array(2).fill('ArrowLeft')));
 
+      browser.pause(150);
       Terra.validates.element('first-cell-in-grid-selected', { selector });
       expect(browser.$$('[role="grid"] [tabIndex="0"]')).toBeElementsArrayOfSize(1);
     });
 
     it('control with home key', () => {
       browser.keys(['Tab']
-        .concat(new Array(4).fill('ArrowDown'))
-        .concat(new Array(3).fill('ArrowRight'))
+        .concat(new Array(2).fill('ArrowDown'))
+        .concat(new Array(2).fill('ArrowRight'))
         .concat(['Control', 'Home', 'Control']));
 
+      browser.pause(150);
       Terra.validates.element('first-non-column-header-cell-selected', { selector });
       expect(browser.$$('[role="grid"] [tabIndex="0"]')).toBeElementsArrayOfSize(1);
     });
 
     it('home key', () => {
       browser.keys(['Tab']
-        .concat(new Array(4).fill('ArrowDown'))
-        .concat(new Array(3).fill('ArrowRight'))
+        .concat(new Array(2).fill('ArrowDown'))
+        .concat(new Array(2).fill('ArrowRight'))
         .concat(['Home']));
 
+      browser.pause(150);
       Terra.validates.element('first-cell-in-row-selected', { selector });
       expect(browser.$$('[role="grid"] [tabIndex="0"]')).toBeElementsArrayOfSize(1);
     });
 
     it('end key', () => {
       browser.keys(['Tab']
-        .concat(new Array(4).fill('ArrowDown'))
-        .concat(new Array(3).fill('ArrowRight'))
+        .concat(new Array(1).fill('ArrowDown'))
         .concat(['End']));
 
+      browser.pause(150);
       Terra.validates.element('last-cell-in-row-selected', { selector });
       expect(browser.$$('[role="grid"] [tabIndex="0"]')).toBeElementsArrayOfSize(1);
     });
 
     it('control with end key', () => {
       browser.keys(['Tab']
-        .concat(new Array(4).fill('ArrowDown'))
-        .concat(new Array(3).fill('ArrowRight'))
         .concat(['Control', 'End', 'Control']));
 
+      browser.pause(150);
       Terra.validates.element('last-cell-in-last-row-selected', { selector });
       expect(browser.$$('[role="grid"] [tabIndex="0"]')).toBeElementsArrayOfSize(1);
     });
 
     it('down arrow cannot navigate outside grid', () => {
       browser.keys(['Tab']
         .concat(['Control', 'End', 'Control'])
-        .concat(new Array(3).fill('ArrowDown')));
+        .concat(new Array(2).fill('ArrowDown')));
 
+      browser.pause(150);
       Terra.validates.element('last-cell-in-last-row-selected', { selector });
       expect(browser.$$('[role="grid"] [tabIndex="0"]')).toBeElementsArrayOfSize(1);
     });
 
     it('right arrow cannot navigate outside grid', () => {
       browser.keys(['Tab']
         .concat(['Control', 'End', 'Control'])
-        .concat(new Array(3).fill('ArrowRight')));
+        .concat(new Array(2).fill('ArrowRight')));
```

Comment:
Same as above:
```suggestion
        .concat(['ArrowDown', ArrowDown']);
```

Reply:
@sdadn - Would you agree that this is more of a programmer preference?  It does not impact performance and makes future changes in count pretty simple, if necessary.

Reply:
I don't think there'll be frequent count changes but since there are no performance issues, that's fine by me.